lib/NormalModuleFactory.js

```diff
@@ -849,6 +849,17 @@ ${err2.stack}`;
 								err.message += `
 ${hints.join("\n\n")}`;
 							}
+
+							// Check if the extension is missing a leading dot (e.g. "js" instead of ".js")
+							const specifiedExtensions = Array.from(
+								resolver.options.extensions
+							);
+							specifiedExtensions.forEach(extension => {
+								if (extension.match(/^[^.]/)) {
+									err.message += `\nDid you miss the leading dot in 'resolve.extensions'? Did you mean '.${extension}' instead of '${extension}'?`;
```

Comment:
Let's improve this and try to write everything in a line, I saw project with a lot of them and it will be un readable

Reply:
@alexander-akait Updated 👍🏻 

<img width="1005" alt="Screenshot 2023-03-12 at 4 15 17 PM" src="https://user-images.githubusercontent.com/46647141/224539794-dfcf99b5-1c38-4502-9495-b71320b39e99.png">